### ISSUE TITLE
Remove red text box from top of each datasets page

### ISSUE THREAD

**Issue by dgamblin3008** [DFE-Digital/npd-find-and-explore#605]
An accessibility audit recommended that the 'the exemptions' link in the red text box that is displayed at the top of each dataset page, i.e. the text box with the title 'Some of these data elements might not be available for research purposes', should have a more detailed contextual description for users of accessibility software.

However, as it's no longer intended that the F&E tool will be used directly in the NPD data application process, the red text box is no longer needed and should be removed from each dataset page.

**Comment by dgamblin3008** [comment]
NB - this ticket has been approved by the Product Owner, Gwen McGill

**Comment by dgamblin3008** [comment]
I have tested this change and confirm it has passed the tests.

**Comment by dgamblin3008** [comment]
Awaiting the result of the accessibility test on 14/09/2020.

**Comment by dgamblin3008** [comment]
The accessibility re-test picked up no issues, so this change is ready to push to production. However, I'll do a final check with Chris Douglass and Martin Johnson, to confirm that they're happy with the change.

**Comment by dgamblin3008** [comment]
Users have checked change. Moving to 'Ready to Release'.

**Comment by dgamblin3008** [comment]
Deployed to production and tested. Awaiting approval by PO, Gwen McGill, before closing the ticket.

**Comment by dgamblin3008** [comment]
PO has agreed to close the ticket.
